pages/common/vala.md

```diff
@@ -0,0 +1,17 @@
+# vala
+
+> Vala code runner.
+> Tutorial: <https://wiki.gnome.org/Projects/Vala/Tutorial>.
+> Doc homepage: <https://valadoc.org/>.
+
+- Run a vala file, with gtk+:
+
+`vala xxx.vala --pkg gtk+-3.0`
```

Comment:
Perhaps we could use tldr-pages' special `{{token_syntax}}` here? It indicates to the reader that they can change something, and tldr clients interpret it specially.
```suggestion
`vala {{path/to/file.vala}} --pkg gtk+-3.0`
```

Reply:
@sbrl, is it worth wrapping `{{gtk+-3.0}}` in tokens as well?

Reply:
Hrm, that's a good idea. Vala seems to have a package system - I'll edit my suggestion.